src/liquid/components/ld-link/ld-link.tsx

```diff
@@ -0,0 +1,62 @@
+import '../../components' // type definitions for type checks and intelliSense
+import { Component, Host, h, Prop } from '@stencil/core'
+import { getClassNames } from 'src/liquid/utils/getClassNames'
+
+/**
+ * @virtualProp ref - reference to component
+ * @virtualProp {string | number} key - for tracking the node's identity when working with lists
+ * @part arrow - adds chevron arrow before link text
```

Comment:
I don't see these parts anywhere in the template. Please have a look at https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes#attr-part